tests/bluetooth/tester/src/audio/btp_bap.c

```diff
@@ -332,7 +332,39 @@ static uint8_t btp_bap_supported_commands(const void *cmd, uint16_t cmd_len,
 
 	/* octet 0 */
 	tester_set_bit(rp->data, BTP_BAP_READ_SUPPORTED_COMMANDS);
-
+	tester_set_bit(rp->data, BTP_BAP_DISCOVER);
+	tester_set_bit(rp->data, BTP_BAP_SEND);
+
+#if defined(CONFIG_BT_BAP_BROADCAST_SOURCE) || defined(CONFIG_BT_BAP_BROADCAST_SINK)
```

Comment:
Please is `IS_ENABLED` instead. 

The selection of the bits also needs to be more fine-tuned: i.e. `BTP_BAP_BROADCAST_SOURCE_SETUP` should not be set if `CONFIG_BT_BAP_BROADCAST_SOURCE=n` and `CONFIG_BT_BAP_BROADCAST_SINK=y`. Same applies for several other bits here, and some also refer to scan delegator or broadcast assistant as well

Reply:
That is the way they are defined below in the bap_handler. But I'll remove all the supported commands that falls under the `#if`. They can rather be added when supported commands are implemented.